test/runtests.jl

```diff
@@ -1,27 +1,30 @@
 module PlotsTests
 
+using Pkg
+Pkg.add(PackageSpec(url="https://github.com/tkf/PlotReferenceImages.jl"))
```

Comment:
Surely you don't want to use my fork :)

Reply:
Oh, thanks for taking such a close look!